cla_backend/apps/call_centre/tests/api/test_case_api.py

```diff
@@ -257,20 +257,44 @@ def response_keys(self):
 
     def test_patch_operator_notes_allowed(self):
         """
-        Test that provider cannot post provider notes
+        Test that operator can post provider notes at max limit
         """
         self.assertEqual(CaseNotesHistory.objects.all().count(), 0)
+
+        max_character_limit = "A" * 10000
```

Comment:
It may be worth changing this line to read
```
max_character_limit = "A" * CaseSerializer.provider_notes.max_length
```
The CaseSerializer is already imported into this module so this would allow you to remove the magic number incase the limit needs changing in the future.

Reply:
Ah darn! Of course, this is better. I've made the changes